tests/phpunit/Checker/Default_Check_Repository_Tests.php

```diff
@@ -116,14 +116,11 @@ public function test_get_checks_returns_checks_via_slug() {
 		$this->repository->register_check( 'static_check', $static_check );
 		$this->repository->register_check( 'runtime_check', $runtime_check );
 
-		$this->assertSame( array( $static_check ), $this->repository->get_checks( Check_Repository::TYPE_ALL, array( 'static_check' ) ) );
-	}
-
-	public function test_get_checks_throws_exception_for_invalid_check_slug() {
```

Comment:
Check above comment.

Reply:
Let's discuss this in https://github.com/WordPress/plugin-check/pull/315#discussion_r1378399550. If we decide to keep the exception, I'll bring back the test.